.github/actions/build/action.yml

```diff
@@ -91,25 +89,50 @@ runs:
         registry: ghcr.io
         username: ${{ github.repository_owner }}
         password: ${{ inputs.ghcr-token }}
+    - name: Build env
+      id: build_env
+      shell: bash
+      run: |
+        if [ "${{ github.event_name }}" == "pull_request" ]
+        then
+            echo "::set-output name=platforms::linux/amd64"
+            echo "::set-output name=push::false"
+            echo "::set-output name=load::true"
+        else
+            echo "::set-output name=platforms::linux/amd64,linux/arm64"
```

Comment:
Please be aware of that not all builds support multi-platforms.

I suggest to set the platforms using `${{ inputs.platforms }}`